django_advanced_queries/covid_19/models.py

```diff
@@ -35,26 +38,61 @@ def __unicode__(self):
         return repr(self)
 
 
+class PersonManager(models.Manager):
+    def get_sick_persons(self):
+        results_subquery = MedicalExaminationResult.objects.filter(
+            patient__person=OuterRef('pk')).order_by('-time')
+
+        return self.annotate(
+            last_result=Subquery(
+                results_subquery.values('result')[:1])).exclude(
+            last_result__in=[MedicalExaminationResult.RESULT_DEAD,
+                             MedicalExaminationResult.RESULT_HEALTHY])
+
+    def persons_with_multiple_jobs(self, jobs=[]):
```

Comment:
Avoid setting instance of something as default value - it defined once.
```
def a(arg=[]):
    arg.append('a')
    return arg
print(a())  # ['a']
print (a())  # ['a', 'a']

```
https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists